packages/odyssey/src/scss/abstracts/_variables.scss

```diff
@@ -14,7 +14,7 @@
 
 // Establish Type Scale
 
-$body-font-family: 'Proxima Nova', '-apple-system',  'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif;
+$body-font-family: 'Whyte', '-apple-system',  'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif;
```

Comment:
Whats the story with this? Why include it if we arent ready to include the font? Why not just leave the base system font?

Reply:
Responded async, but circling back:

We can't distribute Whyte via Odyssey, but we do want it to be the preferred font when available.

For Okta products and properties that our license covers, UI Core has a separate fonts package available.